plugins/outputs/opensearch/sample.conf

```diff
@@ -0,0 +1,103 @@
+# Configuration for OpenSearch to send metrics to.
+[[outputs.OpenSearch]]
+  ## URLs
+  ## The full HTTP endpoint URL for your OpenSearch instance. Multiple URLs can
+  ## be specified as part of the same cluster, but only one URLs is used to
+  ## write during each interval.
+  urls = ["http://node1.os.example.com:9200"]
+
+  ## Index Name
+  ## The target index for metrics (OpenSearch will create if it not exists).
+  ## You can use the date specifiers below to create indexes per time frame.
+  ## The metric timestamp will be used to decide the destination index name
+  ##   %Y - year (2016)
+  ##   %y - last two digits of year (00..99)
+  ##   %m - month (01..12)
+  ##   %d - day of month (e.g., 01)
+  ##   %H - hour (00..23)
+  ##   %V - week of the year (ISO week) (01..53)
+  ## For example: telegraf-%Y.%m.%d would set it to telegraf-2006-01-02
```

Comment:
This is still the old way, please update.

Reply:
updated